nupic/algorithms/anomaly_likelihood.py

```diff
@@ -106,11 +112,7 @@ def __init__(self,
     self._probationaryPeriod = claLearningPeriod + estimationSamples
     self._claLearningPeriod = claLearningPeriod
 
-    # How often we re-estimate the Gaussian distribution. The ideal is to
-    # re-estimate every iteration but this is a performance hit. In general the
-    # system is not very sensitive to this number as long as it is small
-    # relative to the total number of records processed.
-    self._reestimationPeriod = 100
+    self._reestimationPeriod = reestimationPeriod
```

Comment:
do we need to update __setstate__ to handle old saved instances of this? Or is it not being saved by anyone yet? Probably better to be safe.